targets/targets.json

```diff
@@ -7474,6 +7895,8 @@
             "CYBSP_WIFI_CAPABLE"
         ],
         "device_name": "CY8C624ABZI-D44",
+        "mbed_rom_start": "0x10000000",
+        "mbed_rom_size": "0x200000",
         "mbed_ram_start": "0x08002000",
```

Comment:
Is this the right start address? `index.json` says `134217728`, which is `0x8000000`. Maybe we are pretending the ROM starts `0x2000` above that to store a bootloader or for some other reason.

The size is also weird looking. `index.json` says `0x100000`, but we say `0x0FD800`, which is `0x2800` apart, not even matching the `0x2000` offset mentioned in the previous paragraph.

The same issues would apply to any `CY8C624ABZI-D44` data in `targets.json` (not just where I've commented here).